README.md

```diff
@@ -61,7 +61,7 @@ place after an MSC has been accepted, not as part of a proposal itself.
 
 1. Install the extended version (often the OS default) of Hugo:
    <https://gohugo.io/getting-started/installing>. Note that at least Hugo
-   v0.110.0 is required.
+   v0.113.0 is required.
```

Comment:
Is it though? As I said in the chat, there's no need to bump this unless we're aware of problems on v0.110.0.

Reply:
The idea here is to have the same version as in config.toml. The goal of both is to declare the minimum supported version of Hugo for the website.

Reply:
What I'm saying is either we need to downgrade the version in config.toml, or we need to upgrade it in the README.

Reply:
Right, sorry, I misunderstood the purpose of the config.toml setting.